frontend/src/components/header/Header.tsx

```diff
@@ -334,23 +343,38 @@ export default function Header({
     }
   };
 
-  const logo = getLogo();
-
-  const getLogoLink = () => {
-    if (hubUrl) {
-      return `${localePrefix}/hubs/${hubUrl}`;
+  const makeLogoLink = () => {
+    let logoLink = "/";
+    if (isLandingPage || (isHubPage && hubUrl)) {
+      logoLink += `hubs/${hubUrl?.toLowerCase() || ""}`;
     }
-    return `${localePrefix}/`;
+    return logoLink;
   };
-  const logoLink = getLogoLink();
+
+  const logo = getLogo();
+  const logoLink = makeLogoLink();
+  const [hideHeader, setHideHeader] = useState(false);
+  const [lastScrollY, setLastScrollY] = useState(0);
+
+  // const getLogoLink = () => {
+  //   if (hubUrl) {
+  //     return `${localePrefix}/hubs/${hubUrl}`;
+  //   }
+  //   return `${localePrefix}/`;
+  // };
+  // const logoLink = getLogoLink();
 
   return (
     <Box
       component="header"
-      className={`${classes.root} ${className} ${!noSpacingBottom && classes.spacingBottom}`}
+      className={`${classes.root} ${
+        isLandingPage ? classes.landingPageNavColor : ""
+      } ${className} ${!noSpacingBottom && classes.spacingBottom} ${
+        hideHeader ? classes.hideHeader : ""
+      }`}
     >
       <Container className={classes.container}>
-        <Link href={logoLink} className={classes.logoLink} underline="hover">
+        <Link href={localePrefix + logoLink} className={classes.logoLink} underline="hover">
```

Comment:
Wouldn't it be easier to directly add the localePrefix in the `makeLogoLink` function incase we also use it in another place?